src/containers/pages/MDAPoint/ClientUploadStatus/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/ClientUploadStatus Matches snapshot 1`] = `
+<UploadStatus
+  uploadFile={File {}}
+>
+  <p>
+    loading...
+  </p>
+</UploadStatus>
+`;
```

Comment:
This snapshot tells us that this component never loads successfully.  Is this expected?